### DISCUSSION TITLE
Can't create new world on raspberry pi

### DISCUSSION THREAD

**Discussion by terraria-enthusiast** [Pryaxis/TShock#2161] [Q&A] [answered]
I am trying to make a tshock server for raspberry pi, but whenever I try to create the server using mono ./TerrariaServer.exe it takes me through all the steps, but when I have finished selecting all of my world preferences it starts creating the world, and usually at about 30-50% it says 'killed' and stops. When I run it again, it says something about it being closed the wrong way, and then I have to repeat the same process again. If anyone could tell me how to fix it that would be awesome, but I also already have a world I would prefer to use, so if anyone could tell me how to import a world from steam on widows to tshock on raspberry pi then that would also be great as well.

**Comment by Syshein** [comment, marked as answer]
You can copy your world file to your raspberry pi and tell TShock where to find via command line parameters. You can look it up on the documentation:
https://tshock.readme.io/docs/command-line-parameters

**Reply by terraria-enthusiast** [reply]
Thank you! That was a huge help.
